### DISCUSSION TITLE
Welcome to qelectrotech-source-mirror Discussions!

### DISCUSSION THREAD

**Discussion by scorpio810** [qelectrotech/qelectrotech-source-mirror#105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arqeco** [comment]
I am just discovering QElectroTech and would like to say thank you for it

**Reply by scorpio810** [reply]
Thanks

**Reply by thetube2011** [reply]
Bonjour Qelectrotech, merci pour votre travail.
Je suis prof en maintenance et tente depuis deux jours d'enregistrer mon travail sans succèes. Au moment de l'enregistrement ou exportation Qelec se ferme 
Par ailleurs, lorsque j'essaie de m'enregistrer comme nouveau membre sur le forum je suis considéré comme un bot

**Reply by scorpio810** [reply]
Bonjour thetube2011,

Avast est installé sur ton PC?
https://qelectrotech.org/forum/viewtopic.php?id=2421

https://qelectrotech.org/forum/viewtopic.php?id=2416

https://qelectrotech.org/bugtracker/view.php?id=282
https://qelectrotech.org/bugtracker/view.php?id=283


Pour le forum envoie moi un email et j'essayerai de te rajouter manuellement sur le forum.
https://qelectrotech.org/contact.html

Laurent

**Reply by thetube2011** [reply]
Merci pour les liens, pas de Avast mais AVG, je suis sous Win 11. et j'ai la version V 0.90 

Romain

**Reply by scorpio810** [reply]
Avast ou AVG crashe le logiciel d’après le retour des utilisateurs

https://qelectrotech.org/forum/viewtopic.php?pid=17769#p17769

**Reply by thetube2011** [reply]
En effet, après désinstallation de AVG le Qelec me laisse enregistrer

**Reply by thetube2011** [reply]
Qelectro semble bien fonctionner, je verrai plus tard comment l'installer sur une machine dédiée. En attendant je vais commencer l'apprentissage et faire une petite donation, Merci !

**Reply by scorpio810** [reply]
Super, je te conseilles néanmoins plutôt la version 0.100.0 en développement.
Il y a quelques bugs sur la 0.9 qui ont été résolu dernièrement sur la 0.100.0 sur la branche https://git.tuxfamily.org/qet/qet.git/log/?h=0.9.1

**Comment by arqeco** [comment]
I was looking for the history of QElectroTech

**Reply by scorpio810** [reply]
https://github.com/qelectrotech/qelectrotech-source-mirror#story

**Comment by ccarlo88** [comment]
QElectroTech is one of the open sources projects which impresses me most. It is not only really good but it is the best SW for electrical schematic which I have ever found. Even some proprietary software which I tried years ago were not as good as QET. Sad that I don't have a project to try the cable parts and all other features the moment.
**Amazing** job done, congratulations and thank you for this masterpiece!

**Reply by scorpio810** [reply]
Thanks. ;-)

**Comment by thetube2011** [comment]
Bonjour,

Mon voici mon ad email

Le dim. 12 mars 2023 à 09:13, Laurent Trinques ***@***.***> a
écrit :

**Comment by thetube2011** [comment]
Daccord, je l'installerai

A bientôt et merci pour ce super outil qui plus est gratuit c'est vraiment
cool

Le dim. 12 mars 2023 à 12:27, Laurent Trinques ***@***.***> a
écrit :

**Comment by thetube2011** [comment]
Bonjour Laurent,

Je viens de faire une donation modeste parce que je suis un peu juste ces
temps ci. Je tâcherai de faire mieux la prochaine fois

Merci

Le dim. 12 mars 2023 à 12:27, Laurent Trinques ***@***.***> a
écrit :

**Reply by scorpio810** [reply]
Merci Romain de ton soutient. ;-)
